src/firebase.ts

```diff
@@ -16,4 +16,4 @@ const firebaseConfig = {
 export const app = initializeApp(firebaseConfig)
 export const auth = getAuth()
 export const db = getFirestore()
-export const colRef = collection(db, 'users')
```

Comment:
What's happened with the users collection, did you delete it?
I think it's good to keep users in one collection and boards in another (despite the firebase/auth user object).

Reply:
Like so:
registered user has the firebase/auth user object, it's `id` is used to link the authenticated user with your user object from `users` collection in which boardsIds are linked. This way when user is loaded you can get theirs boards. WDYT?

Reply:
>What's happened with the users collection, did you delete it?

Yes, I don't need it.

>registered user has the firebase/auth user object, it's id is used to link the authenticated user with your user object from users collection in which boardsIds are linked. This way when user is loaded you can get theirs boards. WDYT?

This was my previous approach to this data structuring and you said it was wrong. Also, it didn't work, as mentioned [here](https://github.com/Kepmon/task-manager/pull/10#issue-1805753381). After (I hope) reading this, you said I should have the `users` and `boards` collections as separate ones and you told me to implement that. I started doing so, but at some point I realized I didn't actually need the `users` collection because the only thing I care about is the user's ID, which I can grap from this `onAuthStateChanged` function and then use for filtering all boards from the `boards` collection, to get the user's data.